.github/workflows/integration_test.yml

```diff
@@ -15,13 +15,13 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install pytest
-          pip install -r requirements.txt
+          pip install -r dev-requirements.txt
```

Comment:
Could we use a single `requitements.txt`file ? What is the purpose of having a second `dev-requirements.txt`?

Reply:
to specified extra-package for dev... like pytest-cov, sphinx for docs ...
,